src/ai/backend/manager/idle.py

```diff
@@ -307,24 +305,133 @@ class UtilizationIdleChecker(BaseIdleChecker):
     """
 
     name: ClassVar[str] = "utilization"
-    default_cpu_util_threshold: ClassVar[float] = 30.0
-    default_accelerator_util_threshold: ClassVar[float] = 10.0
-    default_initial_grace_period: ClassVar[float] = 300.0  # allow first 5 minutes to be idle
+    cpu_util_series: List[float] = []
+    mem_util_series: List[float] = []
 
-    async def populate_config(self, config: Mapping[str, Any]) -> None:
-        pass
+    idle_timeout: timedelta
 
-    async def check_session(self, session: Row, dbconn: SAConnection) -> bool:
-        # last_stat = session['last_stat']
-        # TODO: implement
-        return True
+    _config_iv = t.Dict(
+        {
+            t.Key("threshold", default="10m"): tx.TimeDuration(),
+        }
+    ).allow_extra("*")
```

Comment:
I think this `_config_iv` should check all the essential Etcd items under `config/idle/checkers/utilization`.

Reply:
I don't have knowledge about what kind of type is suitable for the execution variables. Is it custom defined type or float?
Also, I use None type for checking available resources to construct boolean conditions since etcd returns None if there is no specified resource.

Reply:
I added `_config_iv` for utilization idle checker class.